site/_layouts/sprint.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
```

Comment:
Why is this page sprint.md if it's all html?  Is that a jekyll thing?

Reply:
Addressed